samcli/lib/hook/hook_wrapper.py

```diff
@@ -191,7 +191,7 @@ def get_available_hook_packages_ids() -> List[str]:
     LOG.debug("Return available internal hook packages")
     hook_packages_ids = []
     for child in INTERNAL_PACKAGES_ROOT.iterdir():
-        if child.is_dir():
+        if child.is_dir() and child.name != "__pycache__":
```

Comment:
is __pycache__ the only one we want to exclude?

Reply:
right now, yes. If we wanted to make this less likely to break, we could just hardcode the package ids here? But we'd have to make sure we keep them up to date.

Reply:
We could also change this to check in a list instead if we wanted to be set up for any future excluded folders, but presumably anything inside of that hook_packages folder is a hook_package right (aside from the cache folder)?

Reply:
let's ignore any name that does not start with [a-z] .. these names will be the hooks packages names we choose, so we can assume that it will always start with small alphabet letters

Reply:
yeah, a list to exclude would be good

Reply:
done